src/main/kotlin/org/rust/cargo/projectSettings/RustProjectSettingsService.kt

```diff
@@ -0,0 +1,33 @@
+package org.rust.cargo.projectSettings
+
+import com.intellij.openapi.components.service
+import com.intellij.openapi.module.Module
+import com.intellij.openapi.project.Project
+import com.intellij.util.messages.Topic
+import org.rust.cargo.toolchain.RustToolchain
+
+interface RustProjectSettingsService {
+    var toolchain: RustToolchain?
+    var autoUpdateEnabled: Boolean
+
+    /*
+     * Show a dialog for toolchain configuration
+     */
+    fun configureToolchain()
```

Comment:
That's an invalid Kotlin-doc. I'll fix those (in this P/R), though let's stick to the valid ones.